Sources/PinLayout.swift

```diff
@@ -22,9 +22,19 @@ import Foundation
 #if os(iOS) || os(tvOS)
     import UIKit
     public typealias PEdgeInsets = UIEdgeInsets
+
+    public extension UIEdgeInsets {
```

Comment:
Please remove the `public` keyword. This should be kept private.

Reply:
In fact you should move these extensions inside files `Extensions/UIView+PinLayout` and `Extensions/NSView+PinLayout`.